help/submit_pdf.md

```diff
@@ -54,6 +54,24 @@ one of these free conversion utilities:
 (Please contact us if this information is out of date or if you can
 suggest other free tools that are better.)
 
+<span id="readable"></span>PDF must be machine readable 
+-------------------------------------------------------
+
+arXiv relies on our submitters to provide PDF which are **machine readable**
+i.e. that the PDF is not encrypted, bitmapped, or otherwise altered to prevent
+text extraction. Providing a machine readable PDF facilitates *discovery and access*. 
+This means that we can build tools around your work to provide
+content discovery (e.g. full text search), and that potential readers can still
+understand its contents using tools such as screen readers. 
+
+arXiv does not accept bitmapped or scanned PDF, even in the case of software provided OCR

```

Comment:
arXiv does not accept bitmapped or scanned PDFs, even in the case of software provided OCR